domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/RawFileInfo.java

```diff
@@ -16,16 +16,29 @@
 
 package org.mobilitydata.gtfsvalidator.domain.entity;
 
+/**
+ * The @code{RawFileInfo} class contains information regarding a file location and expected content (file name)
```

Comment:
If you're referring to the class this Javadoc is for, you can omit the name to make the description shorter. So this would be:

>Contains information regarding a file location and expected content (file name)